features/your_results_frequency_selector.feature

```diff
@@ -1,36 +1,69 @@
 Feature:
-  As an employee, I want to be able to see my results in a frequency different to how I input my salary because I want to understand how the contributions will impact my monthly or weekly take-home pay.
+  In order to understand how the contributions will impact my monthly or weekly take-home pay
+  As an employee
+  I want to be able to see my results in a frequency different to how I input my salary
 
   Background:
     Given I am on the Your Details step
+    And I am a "25" year old "female"
```

Comment:
Will Will's work ever come to an end? I think we need to contribute to the clean up too. Just a small contribution.